scripts/set_planned_maintenance.py

```diff
@@ -0,0 +1,33 @@
+import json
+import os
+import sys
+
+import redis
+
+sys.path.append(os.path.join(os.path.dirname(__file__), os.path.pardir))
+
+from app import MAINTENANCE_MSG  # NOQA
+from app import config  # NOQA
+
+
+try:
+    config_info = getattr(config, os.environ['APP_SETTINGS'])
+except (AttributeError, KeyError):
+    config_info = config.DevelopmentConfig
+
+redis_connection = redis.Redis(host=config_info.REDIS_HOST, port=config_info.REDIS_PORT)
+
+
+def main(custom_message, ttl):
+    message_dict = MAINTENANCE_MSG.copy()
+    message_dict['text'] = message_dict['text'].format(custom_message=custom_message)
+    redis_connection.set(config_info.REDIS_MAINTENANCE_KEY, json.dumps(message_dict))
+    redis_connection.expire(config_info.REDIS_MAINTENANCE_KEY, ttl)
```

Comment:
im not sure setting an expiry for a maintenance window message is useful, as i doubt we'll know how long we're gonna be, it would be easier if it was just a set/unset thing - at least as the default behaviour anyway but allow us to pass in a ttl if we want to use it

Reply:
Updated 👍